### PR TITLE
DAM-7047 Add labels

### PR DESCRIPTION
**Pull request by zlepper**

### CODE REVIEW

**Comment by zlepper** [comment]
@Poiar Ping :)

File: MM/5.9.0/config/media_manager_5/labels/Administration.dcl [hunk 1]
```diff
@@ -3167,3 +3167,258 @@ resource configservice_label administration_tools_analytics_chart_builder_sort_r
...
+  key = 'ADMINISTRATION_TOOLS_ANALYTICS_COLOR_BUILDER_ADD'
+  group = 'administration-tools - analytics-color-builder'
+  product_id = resource.configservice_product.media_manager_5.id
+  default_label_values = [
+    {
+      default_translation = 'Add'
+      language_id = data.language.english.id
+    },
+    {
+      default_translation = 'Add'
```
Comment (Poiar) [inline MM/5.9.0/config/media_manager_5/labels/Administration.dcl]:
```suggestion
      default_translation = 'Tilføj'
```

File: MM/5.9.0/config/media_manager_5/labels/Administration.dcl [hunk 2]
```diff
@@ -3167,3 +3167,258 @@ resource configservice_label administration_tools_analytics_chart_builder_sort_r
...
+  key = 'ADMINISTRATION_TOOLS_ANALYTICS_COLOR_BUILDER_REMOVE'
+  group = 'administration-tools - analytics-color-builder'
+  product_id = resource.configservice_product.media_manager_5.id
+  default_label_values = [
+    {
+      default_translation = 'Remove'
+      language_id = data.language.english.id
+    },
+    {
+      default_translation = 'Remove'
```
Comment (Poiar) [inline MM/5.9.0/config/media_manager_5/labels/Administration.dcl]:
```suggestion
      default_translation = 'Fjern'
```

File: MM/5.9.0/config/media_manager_5/labels/Administration.dcl [hunk 3]
```diff
@@ -3167,3 +3167,258 @@ resource configservice_label administration_tools_analytics_chart_builder_sort_r
...
+  key = 'ADMINISTRATION_TOOLS_ANALYTICS_COLOR_BUILDER_COLORS'
+  group = 'administration-tools - analytics-color-builder'
+  product_id = resource.configservice_product.media_manager_5.id
+  default_label_values = [
+    {
+      default_translation = 'Colors'
+      language_id = data.language.english.id
+    },
+    {
+      default_translation = 'Colors'
```
Comment (Poiar) [inline MM/5.9.0/config/media_manager_5/labels/Administration.dcl]:
```suggestion
      default_translation = 'Farver'
```

File: MM/5.9.0/config/media_manager_5/labels/Administration.dcl [hunk 4]
```diff
@@ -3167,3 +3167,258 @@ resource configservice_label administration_tools_analytics_chart_builder_sort_r
...
+  key = 'ADMINISTRATION_TOOLS_ANALYTICS_COLOR_BUILDER_COLOR'
+  group = 'administration-tools - analytics-color-builder'
+  product_id = resource.configservice_product.media_manager_5.id
+  default_label_values = [
+    {
+      default_translation = 'Color'
+      language_id = data.language.english.id
+    },
+    {
+      default_translation = 'Color'
```
Comment (Poiar) [inline MM/5.9.0/config/media_manager_5/labels/Administration.dcl]:
```suggestion
      default_translation = 'Farve'
```

File: MM/5.9.0/config/media_manager_5/labels/Administration.dcl [hunk 5]
```diff
@@ -3167,3 +3167,258 @@ resource configservice_label administration_tools_analytics_chart_builder_sort_r
...
+  key = 'ADMINISTRATION_TOOLS_ANALYTICS_COLOR_BUILDER_FROM_CUSTOM_PALETTE'
+  group = 'administration-tools - analytics-color-builder'
+  product_id = resource.configservice_product.media_manager_5.id
+  default_label_values = [
+    {
+      default_translation = 'From custom palette'
+      language_id = data.language.english.id
+    },
+    {
+      default_translation = 'From custom palette'
```
Comment (Poiar) [inline MM/5.9.0/config/media_manager_5/labels/Administration.dcl]:
```suggestion
      default_translation = 'Fra brugerdefineret palette'
```

File: MM/5.9.0/config/media_manager_5/labels/Administration.dcl [hunk 6]
```diff
@@ -3167,3 +3167,258 @@ resource configservice_label administration_tools_analytics_chart_builder_sort_r
...
+  key = 'ADMINISTRATION_TOOLS_ANALYTICS_COLOR_BUILDER_FROM_STANDARD_PALETTE'
+  group = 'administration-tools - analytics-color-builder'
+  product_id = resource.configservice_product.media_manager_5.id
+  default_label_values = [
+    {
+      default_translation = 'From standard palette'
+      language_id = data.language.english.id
+    },
+    {
+      default_translation = 'From standard palette'
```
Comment (Poiar) [inline MM/5.9.0/config/media_manager_5/labels/Administration.dcl]:
```suggestion
      default_translation = 'Fra standard palette'
```

File: MM/5.9.0/config/media_manager_5/labels/Administration.dcl [hunk 7]
```diff
@@ -3167,3 +3167,258 @@ resource configservice_label administration_tools_analytics_chart_builder_sort_r
...
+  key = 'ADMINISTRATION_TOOLS_ANALYTICS_COLOR_BUILDER_SINGLE_COLOR'
+  group = 'administration-tools - analytics-color-builder'
+  product_id = resource.configservice_product.media_manager_5.id
+  default_label_values = [
+    {
+      default_translation = 'Single color'
+      language_id = data.language.english.id
+    },
+    {
+      default_translation = 'Single color'
```
Comment (Poiar) [inline MM/5.9.0/config/media_manager_5/labels/Administration.dcl]:
```suggestion
      default_translation = 'Enkelt farve'
```

File: MM/5.9.0/config/media_manager_5/labels/Administration.dcl [hunk 8]
```diff
@@ -3167,3 +3167,258 @@ resource configservice_label administration_tools_analytics_chart_builder_sort_r
...
+  key = 'ADMINISTRATION_TOOLS_ANALYTICS_COLOR_BUILDER_COLOR_SCHEME'
+  group = 'administration-tools - analytics-color-builder'
+  product_id = resource.configservice_product.media_manager_5.id
+  default_label_values = [
+    {
+      default_translation = 'Color scheme'
+      language_id = data.language.english.id
+    },
+    {
+      default_translation = 'Color scheme'
```
Comment (Poiar) [inline MM/5.9.0/config/media_manager_5/labels/Administration.dcl]:
```suggestion
      default_translation = 'Farveskema'
```

File: MM/5.9.0/config/media_manager_5/labels/Administration.dcl [hunk 9]
```diff
@@ -3167,3 +3167,258 @@ resource configservice_label administration_tools_analytics_chart_builder_sort_r
...
+  key = 'ADMINISTRATION_TOOLS_ANALYTICS_CHART_BUILDER_DISPLAY_LEGEND'
+  group = 'administration-tools - analytics-chart-builder'
+  product_id = resource.configservice_product.media_manager_5.id
+  default_label_values = [
+    {
+      default_translation = 'Display legend'
+      language_id = data.language.english.id
+    },
+    {
+      default_translation = 'Display legend'
```
Comment (Poiar) [inline MM/5.9.0/config/media_manager_5/labels/Administration.dcl]:
```suggestion
      default_translation = 'Vis forklaring'
```

File: MM/5.9.0/config/media_manager_5/labels/Administration.dcl [hunk 10]
```diff
@@ -3167,3 +3167,258 @@ resource configservice_label administration_tools_analytics_chart_builder_sort_r
...
+  key = 'ADMINISTRATION_TOOLS_ANALYTICS_CHART_BUILDER_VERTICAL'
+  group = 'administration-tools - analytics-chart-builder'
+  product_id = resource.configservice_product.media_manager_5.id
+  default_label_values = [
+    {
+      default_translation = 'Vertical'
+      language_id = data.language.english.id
+    },
+    {
+      default_translation = 'Vertical'
```
Comment (Poiar) [inline MM/5.9.0/config/media_manager_5/labels/Administration.dcl]:
```suggestion
      default_translation = 'Lodret'
```

File: MM/5.9.0/config/media_manager_5/labels/Administration.dcl [hunk 11]
```diff
@@ -3167,3 +3167,258 @@ resource configservice_label administration_tools_analytics_chart_builder_sort_r
...
+  key = 'ADMINISTRATION_TOOLS_ANALYTICS_CHART_BUILDER_HORIZONTAL'
+  group = 'administration-tools - analytics-chart-builder'
+  product_id = resource.configservice_product.media_manager_5.id
+  default_label_values = [
+    {
+      default_translation = 'Horizontal'
+      language_id = data.language.english.id
+    },
+    {
+      default_translation = 'Horizontal'
```
Comment (Poiar) [inline MM/5.9.0/config/media_manager_5/labels/Administration.dcl]:
```suggestion
      default_translation = 'Vandret'
```

File: MM/5.9.0/config/media_manager_5/labels/Administration.dcl [hunk 12]
```diff
@@ -3167,3 +3167,258 @@ resource configservice_label administration_tools_analytics_chart_builder_sort_r
...
+  key = 'ADMINISTRATION_TOOLS_ANALYTICS_CHART_BUILDER_CHART_ORIENTATION'
+  group = 'administration-tools - analytics-chart-builder'
+  product_id = resource.configservice_product.media_manager_5.id
+  default_label_values = [
+    {
+      default_translation = 'Chart orientation'
+      language_id = data.language.english.id
+    },
+    {
+      default_translation = 'Chart orientation'
```
Comment (Poiar) [inline MM/5.9.0/config/media_manager_5/labels/Administration.dcl]:
```suggestion
      default_translation = 'Diagram-orientering'
```

File: MM/5.9.0/config/media_manager_5/labels/Administration.dcl [hunk 13]
```diff
@@ -3167,3 +3167,258 @@ resource configservice_label administration_tools_analytics_chart_builder_sort_r
...
+  key = 'ADMINISTRATION_TOOLS_ANALYTICS_CHART_BUILDER_TENSION'
+  group = 'administration-tools - analytics-chart-builder'
+  product_id = resource.configservice_product.media_manager_5.id
+  default_label_values = [
+    {
+      default_translation = 'Tension'
+      language_id = data.language.english.id
+    },
+    {
+      default_translation = 'Tension'
```
Comment (Poiar) [inline MM/5.9.0/config/media_manager_5/labels/Administration.dcl]:
```suggestion
      default_translation = 'Spænding'
```

Comment (Poiar) [reply MM/5.9.0/config/media_manager_5/labels/Administration.dcl]: I have not encountered this before

File: MM/5.9.0/config/media_manager_5/labels/Administration.dcl [hunk 14]
```diff
@@ -3167,3 +3167,258 @@ resource configservice_label administration_tools_analytics_chart_builder_sort_r
...
+  key = 'ADMINISTRATION_TOOLS_ANALYTICS_CHART_BUILDER_AUTOMATIC'
+  group = 'administration-tools - analytics-chart-builder'
+  product_id = resource.configservice_product.media_manager_5.id
+  default_label_values = [
+    {
+      default_translation = 'Automatic'
+      language_id = data.language.english.id
+    },
+    {
+      default_translation = 'Automatic'
```
Comment (Poiar) [inline MM/5.9.0/config/media_manager_5/labels/Administration.dcl]:
```suggestion
      default_translation = 'Automatisk'
```

File: MM/5.9.0/config/media_manager_5/labels/Administration.dcl [hunk 15]
```diff
@@ -3167,3 +3167,258 @@ resource configservice_label administration_tools_analytics_chart_builder_sort_r
...
+  key = 'ADMINISTRATION_TOOLS_ANALYTICS_CHART_BUILDER_MAX_VALUE'
+  group = 'administration-tools - analytics-chart-builder'
+  product_id = resource.configservice_product.media_manager_5.id
+  default_label_values = [
+    {
+      default_translation = 'Max value'
+      language_id = data.language.english.id
+    },
+    {
+      default_translation = 'Max value'
```
Comment (Poiar) [inline MM/5.9.0/config/media_manager_5/labels/Administration.dcl]:
```suggestion
      default_translation = 'Maks. værdi'
```

File: MM/5.9.0/config/media_manager_5/labels/Administration.dcl [hunk 16]
```diff
@@ -3167,3 +3167,258 @@ resource configservice_label administration_tools_analytics_chart_builder_sort_r
...
+  key = 'ADMINISTRATION_TOOLS_ANALYTICS_CHART_BUILDER_MIN_VALUE'
+  group = 'administration-tools - analytics-chart-builder'
+  product_id = resource.configservice_product.media_manager_5.id
+  default_label_values = [
+    {
+      default_translation = 'Min value'
+      language_id = data.language.english.id
+    },
+    {
+      default_translation = 'Min value'
```
Comment (Poiar) [inline MM/5.9.0/config/media_manager_5/labels/Administration.dcl]:
```suggestion
      default_translation = 'Min. værdi'
```

**Review by Poiar — CHANGES_REQUESTED** [review]

File: MM/5.9.0/config/media_manager_5/labels/Administration.dcl [hunk 17]
```diff
@@ -3167,3 +3167,313 @@ resource configservice_label administration_tools_analytics_chart_builder_sort_r
...
+  key = 'ADMINISTRATION_TOOLS_ANALYTICS_COLOR_BUILDER_FROM_STANDARD_PALETTE'
+  group = 'administration-tools - analytics-color-builder'
+  product_id = resource.configservice_product.media_manager_5.id
+  default_label_values = [
+    {
+      default_translation = 'From standard palette'
+      language_id = data.language.english.id
+    },
+    {
+      default_translation = 'Fra standard palet'
```
Comment (Poiar) [inline MM/5.9.0/config/media_manager_5/labels/Administration.dcl:3260]:
```suggestion
      default_translation = 'Fra standard palette'
```

File: MM/5.9.0/config/media_manager_5/labels/Administration.dcl [hunk 18]
```diff
@@ -3167,3 +3167,313 @@ resource configservice_label administration_tools_analytics_chart_builder_sort_r
...
+  key = 'ADMINISTRATION_TOOLS_ANALYTICS_COLOR_BUILDER_FROM_CUSTOM_PALETTE'
+  group = 'administration-tools - analytics-color-builder'
+  product_id = resource.configservice_product.media_manager_5.id
+  default_label_values = [
+    {
+      default_translation = 'From custom palette'
+      language_id = data.language.english.id
+    },
+    {
+      default_translation = 'Fra brugerdefineret palet'
```
Comment (Poiar) [inline MM/5.9.0/config/media_manager_5/labels/Administration.dcl:3244]:
```suggestion
      default_translation = 'Fra brugerdefineret palette'
```

File: MM/5.9.0/config/media_manager_5/labels/Administration.dcl [hunk 19]
```diff
@@ -3167,3 +3167,313 @@ resource configservice_label administration_tools_analytics_chart_builder_sort_r
...
+  key = 'ADMINISTRATION_TOOLS_ANALYTICS_CHART_BUILDER_VERTICAL'
+  group = 'administration-tools - analytics-chart-builder'
+  product_id = resource.configservice_product.media_manager_5.id
+  default_label_values = [
+    {
+      default_translation = 'Vertical'
+      language_id = data.language.english.id
+    },
+    {
+      default_translation = 'Vertikal'
```
Comment (Poiar) [inline MM/5.9.0/config/media_manager_5/labels/Administration.dcl:3324]:
```suggestion
      default_translation = 'Lodret'
```

File: MM/5.9.0/config/media_manager_5/labels/Administration.dcl [hunk 20]
```diff
@@ -3167,3 +3167,313 @@ resource configservice_label administration_tools_analytics_chart_builder_sort_r
...
+  key = 'ADMINISTRATION_TOOLS_ANALYTICS_CHART_BUILDER_HORIZONTAL'
+  group = 'administration-tools - analytics-chart-builder'
+  product_id = resource.configservice_product.media_manager_5.id
+  default_label_values = [
+    {
+      default_translation = 'Horizontal'
+      language_id = data.language.english.id
+    },
+    {
+      default_translation = 'Horisontal'
```
Comment (Poiar) [inline MM/5.9.0/config/media_manager_5/labels/Administration.dcl:3340]:
```suggestion
      default_translation = 'Vandret'
```

File: MM/5.9.0/config/media_manager_5/labels/Administration.dcl [hunk 21]
```diff
@@ -3167,3 +3167,313 @@ resource configservice_label administration_tools_analytics_chart_builder_sort_r
...
+  key = 'ADMINISTRATION_TOOLS_ANALYTICS_CHART_BUILDER_CHART_ORIENTATION'
+  group = 'administration-tools - analytics-chart-builder'
+  product_id = resource.configservice_product.media_manager_5.id
+  default_label_values = [
+    {
+      default_translation = 'Chart orientation'
+      language_id = data.language.english.id
+    },
+    {
+      default_translation = 'Graf retning'
```
Comment (Poiar) [inline MM/5.9.0/config/media_manager_5/labels/Administration.dcl:3356]:
```suggestion
      default_translation = 'Graf-retning'
```

File: MM/5.9.0/config/media_manager_5/labels/Administration.dcl [hunk 22]
```diff
@@ -3167,3 +3167,313 @@ resource configservice_label administration_tools_analytics_chart_builder_sort_r
...
+  key = 'ADMINISTRATION_TOOLS_DASHBOARD_TOOL_ADD_EXAMPLE_DASHBOARD'
+  group = 'administration-tools - dashboard-tool'
+  product_id = resource.configservice_product.media_manager_5.id
+  default_label_values = [
+    {
+      default_translation = 'Add example dashboard'
+      language_id = data.language.english.id
+    },
+    {
+      default_translation = 'Tilføj eksempel dashboard'
```
Comment (Poiar) [inline MM/5.9.0/config/media_manager_5/labels/Administration.dcl:3436]:
```suggestion
      default_translation = 'Tilføj eksempel-dashboard'
```

File: MM/5.9.0/config/media_manager_5/labels/Administration.dcl [hunk 23]
```diff
@@ -3167,3 +3167,313 @@ resource configservice_label administration_tools_analytics_chart_builder_sort_r
...
+  key = 'ADMINISTRATION_TOOLS_ANALYTICS_CHART_BUILDER_MAP_PROJECTION'
+  group = 'administration-tools - analytics-chart-builder'
+  product_id = resource.configservice_product.media_manager_5.id
+  default_label_values = [
+    {
+      default_translation = 'Map projection'
+      language_id = data.language.english.id
+    },
+    {
+      default_translation = 'Kort projektion'
```
Comment (Poiar) [inline MM/5.9.0/config/media_manager_5/labels/Administration.dcl:3452]:
```suggestion
      default_translation = 'Kort-projektion'
```

**Review by Poiar — CHANGES_REQUESTED** [review]

**Review by Poiar — APPROVED** [review]

**Pull request merged**